### ISSUE TITLE
duplicate "cannot resolve module" error when trying to run non-existent script

### ISSUE THREAD

**Issue by MarkTiedemann** [denoland/deno#2402]
```
❯ deno run does-not-exist.ts
Cannot resolve module "file:///<redacted>/does-not-exist.ts" from "."
Cannot resolve module "file:///<redacted>/does-not-exist.ts" from "."
```

**Comment by ry** [comment]
It would also be great if it gave a line number or something more informative...

**Comment by kitsonk** [comment]
The work I am doing on errors/diagnostics will allow us to raise an error that will log out with as much info as we would like to provide. It would be good to move everything informing the user of something over to those structures.

**Comment by bartlomieju** [comment]
Fixed in #2587

CC @ry
